andrew/euler/euler_22.py

```diff
@@ -0,0 +1,29 @@
+'''
+Using names.txt (right click and 'Save Link/Target As...'), a 46K text file containing over five-thousand first names, begin by sorting it into alphabetical order. Then working out the alphabetical value for each name, multiply this value by its alphabetical position in the list to obtain a name score.
+
+For example, when the list is sorted into alphabetical order, COLIN, which is worth 3 + 15 + 12 + 9 + 14 = 53, is the 938th name in the list. So, COLIN would obtain a score of 938 × 53 = 49714.
+
+What is the total of all the name scores in the file?
+'''
+
+import numpy as np
+
+def length_of_names(name):
+    num = [ord(name[alphabet]) - 64 for alphabet in range(0, len(name))]
+    return sum(num)
+
+def read_data(filename):
```

Comment:
All file operations should use the python context manager.

See: https://www.pythonforbeginners.com/files/with-statement-in-python